packages/mobile/src/tokens/saga.test.ts

```diff
@@ -29,6 +49,20 @@ const firebaseTokenInfo: StoredTokenBalance[] = [
     decimals: 18,
     balance: null,
   },
+}
+
+const fetchBalancesResponse = [
+  {
+    tokenAddress: poofAddress,
+    balance: (5 * Math.pow(10, 18)).toString(),
```

Comment:
nit: feel free to skip ;)
```suggestion
    balance: (5 * 10**18).toString(),
```